### DISCUSSION TITLE
QSM from 6-echo Siemens images

### DISCUSSION THREAD

**Discussion by rosenph** [QSMxT/QSMxT#71] [General]
Pleased to say that the 1-echo swi produced good QSM images. However, when I tried the 6-echo sequence on the Siemens, the QSM processing mysteriously and consistently failed after about 2-3 min. Attached is the screen grab from the Neurodesk output. **LINK REDACTED**

I have uploaded the 6-echo sequence before I changed the file extension to IMA. There are two directories per zip file. I'd be very grateful if you help us out.

**LINKS REDACTED**

**Comment by astewartau** [comment]
Discussed further in https://github.com/QSMxT/QSMxT/issues/72 :)
